packages/quantic/force-app/main/default/lwc/quanticTab/quanticTab.js

```diff
@@ -45,7 +45,17 @@ export default class QuanticTab extends LightningElement {
    * @api
    * @type {boolean}
    */
-  @api clearFiltersOnTabChange = false;
+  @api get clearFiltersOnTabChange() {
+    return this._clearFiltersOnTabChange;
+  }
+  set clearFiltersOnTabChange(value) {
+    // Explicit boolean conversion to handle string values like "false" or "true"
+    this._clearFiltersOnTabChange =
+      value === true ||
+      // @ts-ignore
+      (typeof value === 'string' && value.toLowerCase() === 'true');
+  }
+  _clearFiltersOnTabChange = false;
```

Comment:
This is where I am not certain we should change that behavior

<img width="844" height="851" alt="Image" src="https://github.com/user-attachments/assets/243a1ea4-63c3-4e97-847c-17cf4b784727" />

By standard, `attribute="false"` and `attribute="true"` is the *same* thing, and they are both considered `truthy` both in raw HTML/JS, but also in LWC.

Reply:
The "real" way to have a boolean value in html is this:

`<c-component clear-filters-on-tab-change>` <- `clearFiltersOnTabChange==true`
`<c-component>`                                                 <- `clearFiltersOnTabChange==false`

That's it.